domain/domain.go

```diff
@@ -456,6 +456,30 @@ func (do *Domain) infoSyncerKeeper() {
 	}
 }
 
+func (do *Domain) topologySyncerKeeper() {
+	defer do.wg.Done()
+	ticker := time.NewTicker(infosync.TopologyTimeToRefresh)
```

Comment:
Do we need to recover in domain like infoSyncer?

```
defer recoverInDomain("infoSyncerKeeper", false)
```

Reply:
Sorry, I didn't notice that. I add it now